server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/DeletedMessagesVaultTest.java

```diff
@@ -602,6 +609,67 @@ public void vaultExportShouldResponseIdempotentSideEffect() throws Exception {
         }
     }
 
+    @Test
+    public void vaultPurgeShouldDeleteExpiredMessageThenExportWithNoEntry() throws Exception {
+        Instant now = Instant.now();
+        when(clock.instant())
+            .thenReturn(now, now.plus(366, ChronoUnit.DAYS));
```

Comment:
extract `now.plus(366, ChronoUnit.DAYS)` to an `Instant oneDayAfterExpiration` helps me understand it better.

And you can replace 366 by `RetiontionConfiguration.getRetentionPeriod().plusDays(1)`